domains/transfer-agreements/TransferAgreements.API/API/Controllers/TransferAgreementsController.cs

```diff
@@ -170,17 +175,16 @@ public async Task<ActionResult<EditTransferAgreementEndDate>> EditEndDate(Guid i
     }
 
     [ProducesResponseType(typeof(string), 200)]
+    [ProducesResponseType(typeof(string), 500)]
```

Comment:
Why is it added that it is expected that the endpoint returns 500 internal server error?

Reply:
Because I am bad at coding :) I'll remove it.

Reply:
Removed